### PR TITLE
Added creation of socket directory

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Added a configuration file for create the socket directory during
boot process.

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@nthomas-redhat @brainfunked @shtripat please review

File: tendrl-node-agent.service [hunk 1]
```diff
@@ -9,7 +9,6 @@ ExecReload=/bin/kill -HUP $MAINPID
 KillMode=process
 Restart=on-failure
 PrivateTmp=true
-RuntimeDirectory=tendrl
```
Comment (GowthamShanmugam) [inline tendrl-node-agent.service]: i added this line for create tendrl directory at runtime, now we are creating directory by systemd so this is not required

File: etc/tendrl/node-agent/node-agent-socket.conf.sample [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Create volatile directory for node-agent socket
+
+d /run/tendrl 0755 root root -
```
Comment (shtripat) [inline etc/tendrl/node-agent/node-agent-socket.conf.sample]: `/run/tendrl` or `/var/run/tendrl` ?

Comment (GowthamShanmugam) [inline etc/tendrl/node-agent/node-agent-socket.conf.sample]: @shtripat /var/run/tendrl,  i will change that comment statement :)

**Comment by brainfunked** [comment]
@TimothyAsir Please take the socket creation under systemd.

**Comment by TimothyAsir** [comment]
Sent a patch to manage by systemd -> https://github.com/Tendrl/node-agent/pull/219

**Comment by GowthamShanmugam** [comment]
@brainfunked @shtripat This patch is replaced by @TimothyAsir PR so i am closing this pull request

**Pull request closed without merging**